pkg/storage/allocator_scorer_test.go

```diff
@@ -703,51 +704,51 @@ func TestDiversityRemovalScore(t *testing.T) {
 		{
 			name: "four existing replicas",
 			expected: map[roachpb.StoreID]float64{
-				testStoreUSa15:  1.0 / 4.0,
-				testStoreUSa1:   1.0 / 4.0,
-				testStoreUSb:    1.0 / 2.0,
-				testStoreEurope: 1,
+				testStoreUSa15:  1.0 / (1.0 + (2.0*(0.5+1.0+1.0))/6.0),
```

Comment:
With how complicated these numbers are getting, it may be better to just specify the expected ordering of diversity scores for the replicas (e.g. `USa15 > USa1 > USb > Europe`)

Reply:
Yeah, that's better idea.